### ISSUE TITLE
Is there a way to implement dns-prefetch tags?

### ISSUE THREAD

**Issue by sergehf** [tannerdolby/eleventy-plugin-metagen#17]
Hello again,

Is there a way to include dns-prefetch tags using metagen? We need to include them after the default charset, http-equiv, and viewport tags, but before everything else. Not seeing a way to do this and would appreciate some insight!

**Comment by tannerdolby** [comment]
Hello @sergehf!

Prior to v1.6.0, this wasn't supported in `metagen`.

I have now added support for `<link rel="dns-prefetch">` as well as `<link rel="preconnect">` as they can commonly be paired together for DNS lookups/resolutions. Thank you for raising this issue! 

The plugin now supports two new fields `dns_prefetch`, and `preconnect` in v1.6.0 which can accept two cases, a `string` or an array of strings:

1. A string i.e. `{% metagen ..., dns_prefetch="https://example.com" %}`
2. Or as an array of `href` value strings for defining multiple "dns-prefetch" or "preconnect" tags. 
```
{% metagen 
   ...,
   dns_prefetch=["https://example.com", "https://foo.com"]
%}
```
Go ahead and upgrade `metagen` to the latest version `1.6.0`. Let me know if the support I've added works for what your team needs.
